alpha/declcfg/declcfg.go

```diff
@@ -7,9 +7,9 @@ import (
 )
 
 const (
-	schemaPackage = "olm.package"
-	schemaChannel = "olm.channel"
-	schemaBundle  = "olm.bundle"
+	SchemaPackage = "olm.package"
```

Comment:
Do these consts need to be exported?  I checked existing references, and it's only used w/in the declcfg package, which would not require exporting.